### PR TITLE
Split out IERS Earth Rotation and Leap Second files into separate package

### PR DESCRIPTION
**Pull request by astrofrog**
As discussed in https://github.com/astropy/astropy/issues/14756#issuecomment-1543625765, it might be worth splitting the IERS Earth Rotation and Leap Second files (along with the URLs and ReadMe files) out into a dedicated package, so this is an experimental PR doing this. The experimental astropy-iers-data package can be found here:

https://github.com/astrofrog/astropy-iers-data

The benefits of having dedicated package are:

* If people need the latest files, more recent that what we would have bundled in astropy in the past, we can tell them to simply pip/conda update the astropy-iers-data package. In environments that have restricted internet access this could also mean downloading the source/wheel from PyPI and installing it manually on those machines, which is easier than asking users to download each different file and then get astropy to point at them.
* If the URL of the files breaks at short notice, we can easily release a new astropy-iers-data version and tell users to update, rather than having to do a whole astropy core package release.
* The sdists and wheels for the core package will be smaller as they won't include the data files, and the repository will also not get cluttered over time with changes to these large files (both are 0.8Mb smaller with this PR)
* Because we don't need to worry so much about data size, I've included the IERS A files in astropy-iers-data so that all files can now be accessed without an internet connection
* We could in principle auto-update the package as often as we want - we could even release weekly or daily if we wanted (though need to check how soon we'd reach the PyPI quota)

To be clear, this will essentially have no impact on the average user because astropy-iers-data will just get pulled in as a required dependency of astropy.

The slight downside is maintenance of a separate package, but I have set it up so that it will auto-update the data files (as we did in the core package until now) via PRs and we could always enable auto-releasing each time a PR is merged to minimize overhead.

I've set up a simple CI job in astropy-iers-data which runs the astropy.utils.iers tests, though we could also simply run the full astropy test suite.

We should discuss this, but I think we will want to make sure that every time we do a release of astropy-iers-data we should also (and with an automated PR) bump the minimum version of astropy-iers-data in the core package, which will then be equivalent to updating the files here. I also think we might want to use calendar versioning for the astropy-iers-data package since the date is what is most relevant.

Things to do if we want to go ahead:

* [x] Move the astropy-iers-data package to the astropy organization
* [x] Write simple docs for the astropy-iers-data package (or remove docs/ and populate README)
* [x] Decide on automating releases, set up if needed, and do a first release
* [x] Make sure that the CI in this PR works fine
* [x] Update any docs in the core package

And in a follow-up PR:

* [ ] Automate updating of minimum version pin of astropy-iers-data in astropy core

### CODE REVIEW

File: astropy/utils/iers/iers.py
Comment (pllim) [inline astropy/utils/iers/iers.py:65]: This is going to break a lot of stuff. We need backward compatibility.

**Review by pllim — COMMENTED** [review]
I like `+20 −22,646` ❗

File: astropy/utils/iers/iers.py
Comment (astrofrog) [reply astropy/utils/iers/iers.py:65]: These variables are imported higher up so should work

**Comment by taldcroft** [comment]
@astrofrog - excellent, thanks for diving in to this! I am a totally swamped this week but look forward to checking this out next week.

File: astropy/utils/iers/iers.py
Comment (mhvk) [inline astropy/utils/iers/iers.py:705]: Seeing this example, I think we should indeed add the readme to `astropy_iers_data` (it is a TODO right now), but it might also make sense to hide the package from the user here, i.e., do `from astropy.utils.iers import IERS_B, IERS_B_IAU2000_README`

**Review by mhvk — COMMENTED** [review]
It is nice that this looks easy! The one overall question I have is to what extent we use `astropy_iers_data` in examples; my sense would be to keep it as an implementation detail, i.e., ensure that everything that is needed can be done via `astropy.utils.iers`. (Right now, this only affects the old-style IERS B readme, which was not really exposed; see in-line comment.)

File: astropy/utils/iers/iers.py
Comment (taldcroft) [reply astropy/utils/iers/iers.py:705]: Agreed on both points.

**Review by taldcroft — COMMENTED** [review]
This looks pretty uncontroversial!

Auto-updating `astropy-iers-data` at a reasonable cadence with date-based versions seems good. Likewise keeping the `astropy` requirement synced up also would be a good thing.

**Comment by taldcroft** [comment]
This also needs a doc update:
https://docs.astropy.org/en/stable/install.html#requirements

**Comment by pllim** [comment]
Before this is merged, we need to do org-level stuff:

* Move repo to org
* Expand the repo README to provide reason why it exists and where to find more doc about it in astropy doc
* Create a new team for that repo
* Give correct people the access to repo and PyPI
* Where does this fit on the Team page?

**Comment by astrofrog** [comment]
@pllim - the repo now exists: https://github.com/astropy/astropy-iers-data and I have only pushed basic infrastructure to ``main``. I think it should have the same access teams as the core package since it is just a split out.

@taldcroft @mhvk @pllim - please take a look at https://github.com/astropy/astropy-iers-data/pull/1 which is a PR adding the main content to the astropy-iers-data package. It would be good to have this reviewed properly before pushing to main, and there are a few open questions.

**Comment by pllim** [comment]
So who are the official maintainers for this new package? I need to know who to bug when things break there. 😉 

Preferably at least 2 people who are responsive.

**Comment by astrofrog** [comment]
@pllim - I'm happy to be one of the maintainers

File: astropy/utils/iers/iers.py
Comment (astrofrog) [reply astropy/utils/iers/iers.py:35]: Yes this is temporary, it was so that both my PRs to astropy-jets-data worked

**Comment by weaverba137** [comment]
I've been [busy with other things](https://noirlab.edu/public/news/noirlab2317/), but I'm planning to run some tests on this over the next few days.

**Comment by weaverba137** [comment]
Hopefully this will be helpful to someone: in order to run tests on this branch, I had to make one edit the `setup.cfg` file:
```
install_requires =
    numpy>=1.21
    pyerfa>=2.0
    # Uncomment the following once there is a release
    #astropy-iers-data
    astropy-iers-data@git+https://github.com/astropy/astropy-iers-data.git@v1.0.2023.06.01.10.51.37#egg=astropy-iers-data
    PyYAML>=3.13
    packaging>=19.0
python_requires = >=3.9
```
When I did that, all tests passed except for an unrelated doctest on `astropy/docs/timeseries/io.rst`. That appears to be further than some of the CI tests got on this PR.

**Comment by astrofrog** [comment]
Converting back to draft as I've realized that the astropy.utils.iers docs will need significant updating.

**Comment by pllim** [comment]
If you want this in v6, I would recommend getting this earlier in the cycle than later, so downstream that tests against astropy-dev would have a chance to adapt.

**Comment by astrofrog** [comment]
Yes I plan to work on wrapping this up tomorrow

**Comment by astrofrog** [comment]
I think this is now ready for review and I have done a [first release of astropy-iers-data](https://pypi.org/project/astropy-iers-data/0.2023.6.15.21.10.12/). A couple of notes:

* I think the docs page I have updated here needs a careful look, as things are a little different now we are bundling the IERS A table so the whole page needs a careful read, not just the changes I made (it isn't very long though). In particular I don't understand the IERS package well enough to understand exactly under what conditions a download will be triggered now.
* ``IERS_A.open`` and ``IERS_A.read`` used to work without specifying ``file=`` if and only if a ``finals2000A.all`` file was present in the working directory, and would fail otherwise. Now, as for ``IERS_B``, it will use the bundled version in astropy-iers-data. However, since this is an API change, for now I check if there is a local file called ``finals2000A.all`` and use that if so over the astropy-iers-data version, and have included a deprecation warning to encourage users to specify ``file=`` explicitly if using a local file since I think that is cleaner.
* I have not yet changed the default maximum age for IERS_A as I thought it would be cleaner to do this in a separate PR and think it would be better for one of the astropy.utils.iers maintainers to do this.

**Comment by astrofrog** [comment]
Note that there is a test failure on Python 3.9 32-bit that seems related to the changes here, but I don't understand why that happens (and why only for 32-bit).

**Comment by pllim** [comment]
```
_________________ TestTimeUT1SpecificIERSTable.test_ut1_iers_A _________________

self = <astropy.time.tests.test_ut1.TestTimeUT1SpecificIERSTable object at 0xe287f5f8>

    @pytest.mark.skipif(not HAS_IERS_A, reason="requires IERS_A")
    def test_ut1_iers_A(self):
>       do_ut1_prediction_tst(iers.IERS_A)

astropy/time/tests/test_ut1.py:145: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

iers_type = <class 'astropy.utils.iers.iers.IERS_A'>

    def do_ut1_prediction_tst(iers_type):
        tnow = Time.now()
        iers_tab = iers_type.open()
        tnow.delta_ut1_utc, status = iers_tab.ut1_utc(tnow, return_status=True)
>       assert status == iers.FROM_IERS_A_PREDICTION
E       assert -2 == 2
E        +  where 2 = iers.FROM_IERS_A_PREDICTION

astropy/time/tests/test_ut1.py:27: AssertionError
```

https://github.com/astropy/astropy/blob/f83796357694aeac424eff9b916126f2d7ed5460/astropy/utils/iers/iers.py#L80-L85

I have no idea what is going on but hope this helps...

**Comment by mhvk** [comment]
My suspicion is that the tests were broken but never ran because in most test scenarios one does not have `finals2000A` in the working directory (which is what `HAS_IERS_A` meant until this PR, i.e., it was essentially always `False`; but now it will be `True`).

Given that it fails on 32 bit, it is likely some annoying short vs long integer problem... I'll see if I can figure it out.

**Comment by mhvk** [comment]
Actually, I think the 32 bit is a red herring - it is about running the tests twice (since it fails the second time around). Maybe because a table is read but not cleared...

**Comment by mhvk** [comment]
OK, I can reproduce with (in python, in test environment)
```
from astropy import test
test(package='utils.iers')
test(package='time')
```
(I hope I can also find out where clean-up is not done correctly; comments here mostly in case I run out of time)

**Comment by mhvk** [comment]
OK, looks like the cache was not cleared properly. I pushed a quick fix, though really the tests could do with a bit of a rewrite... since I wrote most of them, I'll just take it as a sign that now I know better :facepalm: Anyway, for another PR...

**Comment by astrofrog** [comment]
Thank you for figuring this out!

**Comment by astrofrog** [comment]
This is ready for a final review!

**Comment by pllim** [comment]
There is a conflict now, so you need to rebase. FYI.

File: astropy/utils/data.py
Comment (mhvk) [inline astropy/utils/data.py:1030]: Should we (eventually) deprecate this?

**Review by mhvk — APPROVED** [review]
@astrofrog - this looks great! One query in-line, but happy to have this in!

File: astropy/utils/data.py
Comment (astrofrog) [reply astropy/utils/data.py:1030]: We could do it here potentially if you like, do I recommend using the variables from ``astropy_iers_data`` instead?

Comment (mhvk) [reply astropy/utils/data.py:1030]: I guess indeed the ideal way would be to use `IERS_B_README` etc. I think it still makes sense to get those from `astropy.utils.iers`, so that we keep `astropy_iers_data` as an implementation detail. What do you think?

Comment (astrofrog) [reply astropy/utils/data.py:1030]: That sounds good

Comment (astrofrog) [reply astropy/utils/data.py:1033]: Added in https://github.com/astropy/astropy/pull/14819/commits/8f469a23107d5ce1114183a0e992d08a0984aa03

Result:

```
In [2]: get_pkg_data_filename('Leap_Second.dat', package='astropy.utils.iers')
WARNING: AstropyDeprecationWarning: Accessing Leap_Second.dat in this way is deprecated, use astropy.utils.iers.IERS_LEAP_SECOND_FILE instead [astropy.utils.data]
Out[2]: '/Users/tom/Dropbox/Code/Astropy/astropy-iers-data/astropy_iers_data/data/Leap_Second.dat'
```

**Review by mhvk — APPROVED** [review]
Nice! That should really help anyone who faces the deprecation warning.

**Comment by mhvk** [comment]
Except of course the tests need to be adjusted to expect a warning...

**Comment by astrofrog** [comment]
I think the tests should be fixed now 🤞

**Comment by astrofrog** [comment]
@mhvk - looks like CI is passing!

@taldcroft - are you happy with this as-is too?

File: docs/whatsnew/6.0.rst
Comment (pllim) [inline docs/whatsnew/6.0.rst:110]: Is there supposed to be a dash or not? Previous mentions above have a dash. I think we should be consistent one way or another. This comment applies to all other similar occurrences.
```suggestion
service (these are the IERS-A, IERS-B, and leap second tables). IERS-A used
to always be automatically downloaded, whereas the IERS-B and leap second
```

**Review by pllim — COMMENTED** [review]
Definitely a fan of `+160 −22,658`!

Are there companion PRs to backport branches (`v5.3.x` and `v5.0.x`) to keep the "update IERS" workflow running but only for those? I think they have diverged enough that I cannot just auto backport the PR for `v5.3.x` to `v5.0.x`.

I think https://docs.astropy.org/en/latest/install.html#requirements also has to be updated to add `astropy-iers-data` to the listing.

As for the API change, I'll leave approvals to `utils.iers` maintainers (which I cannot find on the Team page but I guess it is @mhvk and @eteq? 😆).

**Comment by pllim** [comment]
Might wanna rebase again because devdeps should be green again now unless you introduced new failures.

**Comment by pllim** [comment]
pyinstaller failure looks related.

**Comment by astrofrog** [comment]
Thanks for catching that! Will investigate.

File: docs/utils/iers.rst
Comment (astrofrog) [reply docs/utils/iers.rst:158]: You are right that it is not yet - I've added a recipe here: https://github.com/conda-forge/staged-recipes/pull/23209

File: astropy/utils/data.py
Comment (astrofrog) [reply astropy/utils/data.py:1041]: You are right that it isn't needed

Comment (astrofrog) [reply astropy/utils/data.py:1041]: Black is adding this back when I remove it :shrug:

File: docs/changes/14819.other.rst
Comment (astrofrog) [reply docs/changes/14819.other.rst]: Fixed

**Comment by astrofrog** [comment]
Hopefully the pyinstaller job should be fixed now

**Comment by pllim** [comment]
devdeps failure unrelated. Not sure what is going on with that change log check. But I would withhold approval until after astropy-iers-data is on conda-forge.

**Comment by mhvk** [comment]
Had another look just to see (and learned a bit about pyinstaller, I guess), but this is very nice. Would seem good to wait till `astropy-iers-data` is on conda before merging. A propos such preparation, let me also ping @olebole - this does not seem a problem for Debian at all, but might as well be sure.

**Comment by astrofrog** [comment]
Still waiting on https://github.com/conda-forge/staged-recipes/pull/23209 at this point, it could be a little while as some other Python packages were ready for review a few weeks ago but are still waiting

**Comment by astrofrog** [comment]
See https://github.com/astropy/astropy/pull/15013 for a companion PR to adjust the auto-update workflow (should be merged after this one)

**Comment by olebole** [comment]
@mhvk that is fine for me. It would also allow a simple backport of the data package.

**Comment by astrofrog** [comment]
@pllim - the conda-forge package is now available: https://anaconda.org/conda-forge/astropy-iers-data so I think we can proceed with this?

**Comment by pllim** [comment]
`conda install astropy-iers-data -c conda-forge` is not installing anything for me on Windows with Python 3.11. Maybe wait a few more days just to be sure?

**Comment by astrofrog** [comment]
Huh weird it is there on the conda forge channel - does conda complain about not finding the package?

**Comment by astrofrog** [comment]
See https://anaconda.org/conda-forge/astropy-iers-data/files for the files

**Comment by pllim** [comment]
Let me see... I was letting it run in the background. Took a very long time and now it wants to update a lot of packages. Is this normal? Why is a simple data package so demanding?

```
The following NEW packages will be INSTALLED:

  astropy-iers-data  conda-forge/noarch::astropy-iers-data-0.2023.7.6.22.3.16-pyhd8ed1ab_0
  libexpat           conda-forge/linux-64::libexpat-2.5.0-hcb278e6_1
  libnsl             conda-forge/linux-64::libnsl-2.0.0-h7f98852_0
  libsqlite          conda-forge/linux-64::libsqlite-3.42.0-h2797004_0
  libzlib            conda-forge/linux-64::libzlib-1.2.13-hd590300_5
  pip                conda-forge/noarch::pip-23.1.2-pyhd8ed1ab_0
  python_abi         conda-forge/linux-64::python_abi-3.11-3_cp311
  typing-extensions  conda-forge/noarch::typing-extensions-4.7.1-hd8ed1ab_0
  typing_extensions  conda-forge/noarch::typing_extensions-4.7.1-pyha770c72_0

The following packages will be UPDATED:

  bzip2                   pkgs/main::bzip2-1.0.8-h7b6447c_0 --> conda-forge::bzip2-1.0.8-h7f98852_4
  ca-certificates    pkgs/main::ca-certificates-2023.01.10~ --> conda-forge::ca-certificates-2023.5.7-hbcca054_0
  cython             pkgs/main::cython-0.29.33-py311h6a678~ --> conda-forge::cython-0.29.36-py311hb755f60_0
  filelock           pkgs/main/linux-64::filelock-3.9.0-py~ --> conda-forge/noarch::filelock-3.12.2-pyhd8ed1ab_0
  identify           pkgs/main/linux-64::identify-2.5.5-py~ --> conda-forge/noarch::identify-2.5.24-pyhd8ed1ab_0
  ld_impl_linux-64   pkgs/main::ld_impl_linux-64-2.38-h118~ --> conda-forge::ld_impl_linux-64-2.40-h41732ed_0
  libgcc-ng          pkgs/main::libgcc-ng-11.2.0-h1234567_1 --> conda-forge::libgcc-ng-13.1.0-he5830b7_0
  libgomp              pkgs/main::libgomp-11.2.0-h1234567_1 --> conda-forge::libgomp-13.1.0-he5830b7_0
  libstdcxx-ng       pkgs/main::libstdcxx-ng-11.2.0-h12345~ --> conda-forge::libstdcxx-ng-13.1.0-hfd8a6a1_0
  libuuid              pkgs/main::libuuid-1.41.5-h5eee18b_0 --> conda-forge::libuuid-2.38.1-h0b41bf4_0
  nodeenv            pkgs/main/linux-64::nodeenv-1.7.0-py3~ --> conda-forge/noarch::nodeenv-1.8.0-pyhd8ed1ab_0
  openssl              pkgs/main::openssl-1.1.1t-h7f8727e_0 --> conda-forge::openssl-3.1.1-hd590300_1
  platformdirs       pkgs/main/linux-64::platformdirs-2.5.~ --> conda-forge/noarch::platformdirs-3.8.0-pyhd8ed1ab_0
  pre-commit         pkgs/main/linux-64::pre-commit-3.2.1-~ --> conda-forge/noarch::pre-commit-3.3.3-pyha770c72_0
  python                pkgs/main::python-3.11.0-h7a1cb2a_2 --> conda-forge::python-3.11.4-hab00c5b_0_cpython
  pyyaml              pkgs/main::pyyaml-6.0-py311h5eee18b_1 --> conda-forge::pyyaml-6.0-py311hd4cff14_5
  readline               pkgs/main::readline-8.2-h5eee18b_0 --> conda-forge::readline-8.2-h8228510_1
  setuptools         pkgs/main/linux-64::setuptools-65.5.0~ --> conda-forge/noarch::setuptools-68.0.0-pyhd8ed1ab_0
  sqlite                pkgs/main::sqlite-3.40.1-h5082296_0 --> conda-forge::sqlite-3.42.0-h2c6b66d_0
  tzdata                 pkgs/main::tzdata-2022g-h04d1e81_0 --> conda-forge::tzdata-2023c-h71feb2d_0
  ukkonen            pkgs/main::ukkonen-1.0.1-py311hdb19cb~ --> conda-forge::ukkonen-1.0.1-py311h4dd048b_3
  virtualenv         pkgs/main/linux-64::virtualenv-20.17.~ --> conda-forge/noarch::virtualenv-20.23.1-pyhd8ed1ab_0
  wheel                pkgs/main::wheel-0.37.1-pyhd3eb1b0_0 --> conda-forge::wheel-0.40.0-pyhd8ed1ab_0
  yaml                     pkgs/main::yaml-0.2.5-h7b6447c_0 --> conda-forge::yaml-0.2.5-h7f98852_2
  zlib                    pkgs/main::zlib-1.2.13-h5eee18b_0 --> conda-forge::zlib-1.2.13-hd590300_5

The following packages will be SUPERSEDED by a higher-priority channel:

  _libgcc_mutex           pkgs/main::_libgcc_mutex-0.1-main --> conda-forge::_libgcc_mutex-0.1-conda_forge
  _openmp_mutex          pkgs/main::_openmp_mutex-5.1-1_gnu --> conda-forge::_openmp_mutex-4.5-2_gnu
  certifi            pkgs/main/linux-64::certifi-2023.5.7-~ --> conda-forge/noarch::certifi-2023.5.7-pyhd8ed1ab_0
  cffi               pkgs/main::cffi-1.15.1-py311h5eee18b_3 --> conda-forge::cffi-1.15.1-py311h409f033_3
  cfgv               pkgs/main/linux-64::cfgv-3.3.1-py311h~ --> conda-forge/noarch::cfgv-3.3.1-pyhd8ed1ab_0
  distlib            pkgs/main/linux-64::distlib-0.3.6-py3~ --> conda-forge/noarch::distlib-0.3.6-pyhd8ed1ab_0
  libffi                 pkgs/main::libffi-3.4.2-h6a678d5_6 --> conda-forge::libffi-3.4.2-h7f98852_5
  ncurses                 pkgs/main::ncurses-6.4-h6a678d5_0 --> conda-forge::ncurses-6.4-hcb278e6_0
  pycparser          pkgs/main::pycparser-2.21-pyhd3eb1b0_0 --> conda-forge::pycparser-2.21-pyhd8ed1ab_0
  tk                        pkgs/main::tk-8.6.12-h1ccaba5_0 --> conda-forge::tk-8.6.12-h27826a3_0
  xz                        pkgs/main::xz-5.2.10-h5eee18b_1 --> conda-forge::xz-5.2.6-h166bdaf_0

Proceed ([y]/n)?
```

**Comment by pllim** [comment]
On a clean conda env with Python 3.11, it still pulls in other stuff but does not take as long or as many:

```
  added / updated specs:
    - astropy-iers-data


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    astropy-iers-data-0.2023.7.6.22.3.16|     pyhd8ed1ab_0         1.2 MB  conda-forge
    ca-certificates-2023.5.7   |       hbcca054_0         145 KB  conda-forge
    ld_impl_linux-64-2.40      |       h41732ed_0         688 KB  conda-forge
    libexpat-2.5.0             |       hcb278e6_1          76 KB  conda-forge
    libgcc-ng-13.1.0           |       he5830b7_0         758 KB  conda-forge
    libgomp-13.1.0             |       he5830b7_0         409 KB  conda-forge
    libsqlite-3.42.0           |       h2797004_0         809 KB  conda-forge
    libstdcxx-ng-13.1.0        |       hfd8a6a1_0         3.7 MB  conda-forge
    libuuid-2.38.1             |       h0b41bf4_0          33 KB  conda-forge
    libzlib-1.2.13             |       hd590300_5          60 KB  conda-forge
    ncurses-6.4                |       hcb278e6_0         860 KB  conda-forge
    openssl-3.1.1              |       hd590300_1         2.5 MB  conda-forge
    pip-23.1.2                 |     pyhd8ed1ab_0         1.3 MB  conda-forge
    python-3.11.4              |hab00c5b_0_cpython        29.3 MB  conda-forge
    readline-8.2               |       h8228510_1         275 KB  conda-forge
    setuptools-68.0.0          |     pyhd8ed1ab_0         453 KB  conda-forge
    sqlite-3.42.0              |       h2c6b66d_0         799 KB  conda-forge
    tzdata-2023c               |       h71feb2d_0         115 KB  conda-forge
    wheel-0.40.0               |     pyhd8ed1ab_0          54 KB  conda-forge
    zlib-1.2.13                |       hd590300_5          91 KB  conda-forge
    ------------------------------------------------------------
                                           Total:        43.4 MB

The following NEW packages will be INSTALLED:

  astropy-iers-data  conda-forge/noarch::astropy-iers-data-0.2023.7.6.22.3.16-pyhd8ed1ab_0
  libexpat           conda-forge/linux-64::libexpat-2.5.0-hcb278e6_1
  libnsl             conda-forge/linux-64::libnsl-2.0.0-h7f98852_0
  libsqlite          conda-forge/linux-64::libsqlite-3.42.0-h2797004_0
  libzlib            conda-forge/linux-64::libzlib-1.2.13-hd590300_5

The following packages will be UPDATED:

  bzip2                   pkgs/main::bzip2-1.0.8-h7b6447c_0 --> conda-forge::bzip2-1.0.8-h7f98852_4
  ld_impl_linux-64   pkgs/main::ld_impl_linux-64-2.38-h118~ --> conda-forge::ld_impl_linux-64-2.40-h41732ed_0
  libgcc-ng          pkgs/main::libgcc-ng-11.2.0-h1234567_1 --> conda-forge::libgcc-ng-13.1.0-he5830b7_0
  libgomp              pkgs/main::libgomp-11.2.0-h1234567_1 --> conda-forge::libgomp-13.1.0-he5830b7_0
  libstdcxx-ng       pkgs/main::libstdcxx-ng-11.2.0-h12345~ --> conda-forge::libstdcxx-ng-13.1.0-hfd8a6a1_0
  libuuid              pkgs/main::libuuid-1.41.5-h5eee18b_0 --> conda-forge::libuuid-2.38.1-h0b41bf4_0
  openssl               pkgs/main::openssl-3.0.9-h7f8727e_0 --> conda-forge::openssl-3.1.1-hd590300_1
  readline               pkgs/main::readline-8.2-h5eee18b_0 --> conda-forge::readline-8.2-h8228510_1
  setuptools         pkgs/main/linux-64::setuptools-67.8.0~ --> conda-forge/noarch::setuptools-68.0.0-pyhd8ed1ab_0
  sqlite                pkgs/main::sqlite-3.41.2-h5eee18b_0 --> conda-forge::sqlite-3.42.0-h2c6b66d_0
  wheel              pkgs/main/linux-64::wheel-0.38.4-py31~ --> conda-forge/noarch::wheel-0.40.0-pyhd8ed1ab_0
  zlib                    pkgs/main::zlib-1.2.13-h5eee18b_0 --> conda-forge::zlib-1.2.13-hd590300_5

The following packages will be SUPERSEDED by a higher-priority channel:

  _libgcc_mutex           pkgs/main::_libgcc_mutex-0.1-main --> conda-forge::_libgcc_mutex-0.1-conda_forge
  _openmp_mutex          pkgs/main::_openmp_mutex-5.1-1_gnu --> conda-forge::_openmp_mutex-4.5-2_gnu
  ca-certificates    pkgs/main::ca-certificates-2023.05.30~ --> conda-forge::ca-certificates-2023.5.7-hbcca054_0
  libffi                 pkgs/main::libffi-3.4.4-h6a678d5_0 --> conda-forge::libffi-3.4.2-h7f98852_5
  ncurses                 pkgs/main::ncurses-6.4-h6a678d5_0 --> conda-forge::ncurses-6.4-hcb278e6_0
  pip                pkgs/main/linux-64::pip-23.1.2-py311h~ --> conda-forge/noarch::pip-23.1.2-pyhd8ed1ab_0
  python                pkgs/main::python-3.11.4-h955ad1f_0 --> conda-forge::python-3.11.4-hab00c5b_0_cpython
  tk                        pkgs/main::tk-8.6.12-h1ccaba5_0 --> conda-forge::tk-8.6.12-h27826a3_0
  tzdata                 pkgs/main::tzdata-2023c-h04d1e81_0 --> conda-forge::tzdata-2023c-h71feb2d_0
  xz                         pkgs/main::xz-5.4.2-h5eee18b_0 --> conda-forge::xz-5.2.6-h166bdaf_0


Proceed ([y]/n)?
```

I see nothing amiss in https://github.com/conda-forge/astropy-iers-data-feedstock/blob/main/recipe/meta.yaml so maybe this is just conda black magic I should walk away from. I don't think this behavior blocks the PR.

**Review by pllim — APPROVED** [review]
Let's do a Friday evening merge on breaking changes! 😉 

Thanks, all!

**Pull request merged**
